Dockerfile

```
@@ -1,6 +1,9 @@
 ﻿FROM node:22-trixie-slim AS deps
 WORKDIR /app
 COPY package.json package-lock.json* ./
+RUN apt-get update \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)